README.md

```diff
@@ -1,16 +1,14 @@
 # LiveFixtures
 
-Welcome to your new gem! In this directory, you'll find the files you need to be able to package up your Ruby library into a gem. Put your Ruby code in the file `lib/live_fixtures`. To experiment with that code, run `bin/console` for an interactive prompt.
-
-TODO: Delete this and the text above, and describe your gem
+ActiveRecord::Fixtures are a powerful way of populating data in a db;
+however, its strategy for handling primary keys and associations is
+UNACCEPTABLE for use with a production db. LiveFixtures works around this.
```

Comment:
Great flag, thank you! I've softened the wording, included a bit more context, and provided a link to more info.